content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -202,21 +203,21 @@ inputStreamReader = new InputStreamReader(socket.getInputStream());
 outputStreamWriter = new OutputStreamWriter(socket.getOutputStream());
 ``` 
 
-The Scanner class is used to get input from the user from the console/terminal. System. in is a keyboard input connected input stream system.
+The Scanner class is used to get input from the user from the console/terminal. System.in is a keyboard input connected input stream system.
```

Comment:
Change:

>The Scanner class is used to get input from the user from the console/terminal. System.in is a keyboard input connected input stream system.
>
>
>```java
>Scanner scanner = new Scanner(System.in);
>```

To:

>Client-side, we need to use a Scanner object to get input from the console/terminal.
>
>```java
>Scanner scanner = new Scanner(System.in);
>```
>`System.in` specifies that we are getting the keyboard input.

I added a smoother transition from the previous sentence into this part. I also moved the second sentence after the code snippet for better flow and simplified it.